scripts/v1beta1/push.sh

```diff
@@ -36,72 +36,75 @@ echo "Image tag: ${TAG}"
 
 # Katib core images
 echo -e "\nPushing Katib controller image...\n"
-docker push ${REGISTRY}/katib-controller:${TAG}
+docker push "${REGISTRY}/katib-controller:${TAG}"
 
 echo -e "\nPushing Katib DB manager image...\n"
-docker push ${REGISTRY}/katib-db-manager:${TAG}
+docker push "${REGISTRY}/katib-db-manager:${TAG}"
 
 echo -e "\nPushing Katib UI image...\n"
-docker push ${REGISTRY}/katib-ui:${TAG}
+docker push "${REGISTRY}/katib-ui:${TAG}"
 
 echo -e "\nPushing Katib cert generator image...\n"
-docker push ${REGISTRY}/cert-generator:${TAG}
+docker push "${REGISTRY}/cert-generator:${TAG}"
 
 echo -e "\nPushing file metrics collector image...\n"
-docker push ${REGISTRY}/file-metrics-collector:${TAG}
+docker push "${REGISTRY}/file-metrics-collector:${TAG}"
 
 echo -e "\nPushing TF Event metrics collector image...\n"
-docker push ${REGISTRY}/tfevent-metrics-collector:${TAG}
+docker push "${REGISTRY}/tfevent-metrics-collector:${TAG}"
 
 # Suggestion images
 echo -e "\nPushing suggestion images..."
 
 echo -e "\nPushing hyperopt suggestion...\n"
-docker push ${REGISTRY}/suggestion-hyperopt:${TAG}
+docker push "${REGISTRY}/suggestion-hyperopt:${TAG}"
 
 echo -e "\nPushing chocolate suggestion...\n"
-docker push ${REGISTRY}/suggestion-chocolate:${TAG}
+docker push "${REGISTRY}/suggestion-chocolate:${TAG}"
 
 echo -e "\nPushing hyperband suggestion...\n"
-docker push ${REGISTRY}/suggestion-hyperband:${TAG}
+docker push "${REGISTRY}/suggestion-hyperband:${TAG}"
 
 echo -e "\nPushing skopt suggestion...\n"
-docker push ${REGISTRY}/suggestion-skopt:${TAG}
+docker push "${REGISTRY}/suggestion-skopt:${TAG}"
 
 echo -e "\nPushing goptuna suggestion...\n"
-docker push ${REGISTRY}/suggestion-goptuna:${TAG}
+docker push "${REGISTRY}/suggestion-goptuna:${TAG}"
 
 echo -e "\nPushing optuna suggestion...\n"
-docker push ${REGISTRY}/suggestion-optuna:${TAG}
+docker push "${REGISTRY}/suggestion-optuna:${TAG}"
 
 echo -e "\nPushing ENAS suggestion...\n"
-docker push ${REGISTRY}/suggestion-enas:${TAG}
+docker push "${REGISTRY}/suggestion-enas:${TAG}"
 
 echo -e "\nPushing DARTS suggestion...\n"
-docker push ${REGISTRY}/suggestion-darts:${TAG}
+docker push "${REGISTRY}/suggestion-darts:${TAG}"
 
 # Early stopping images
 echo -e "\nPushing early stopping images...\n"
 
 echo -e "\nPushing median stopping rule...\n"
-docker push ${REGISTRY}/earlystopping-medianstop:${TAG}
+docker push "${REGISTRY}/earlystopping-medianstop:${TAG}"
 
 # Training container images
 echo -e "\nPushing training container images..."
 
 echo -e "\nPushing mxnet mnist training container example...\n"
-docker push ${REGISTRY}/mxnet-mnist:${TAG}
+docker push "${REGISTRY}/mxnet-mnist:${TAG}"
+
+echo -e "\nPushing Tensorflow with summaries mnist training container example...\n"
+docker push "${REGISTRY}/tf-mnist-with-summaries:${TAG}"
```

Comment:
Please update this doc also: https://github.com/kubeflow/katib/blob/master/docs/images-location.md#training-containers-images.

Reply:
It makes sense, thanks! @andreyvelich

Reply:
I addressed in https://github.com/kubeflow/katib/pull/1731/commits/61a8dd85f63aa0843d55c2d8e8f56ba0a9b4c899